config/main.py

```diff
@@ -1902,36 +1929,43 @@ def override_config_table(db, input_config_db, dry_run):
                     fg='magenta')
         sys.exit(1)
 
-    config_db = db.cfgdb
-
-    # Read config from configDB
-    current_config = config_db.get_config()
-    # Serialize to the same format as json input
-    sonic_cfggen.FormatConverter.to_serialized(current_config)
+    cfgdb_clients = db.cfgdb_clients
 
-    updated_config = update_config(current_config, config_input)
+    for ns, config_db in cfgdb_clients.items():
+        # Read config from configDB
+        current_config = config_db.get_config()
+        # Serialize to the same format as json input
+        sonic_cfggen.FormatConverter.to_serialized(current_config)
 
-    yang_enabled = device_info.is_yang_config_validation_enabled(config_db)
-    if yang_enabled:
-        # The ConfigMgmt will load YANG and running
-        # config during initialization.
-        try:
-            cm = ConfigMgmt()
-            cm.validateConfigData()
-        except Exception as ex:
-            click.secho("Failed to validate running config. Error: {}".format(ex), fg="magenta")
-            sys.exit(1)
+        if multi_asic.is_multi_asic():
+            ns_config_input = config_input[ns]
+            generate_sysinfo(ns_config_input, ns)
+        else:
+            ns_config_input = config_input
+            generate_sysinfo(ns_config_input)
```

Comment:
>generate_sysinfo [](http://example.com/codeflow?start=12&length=16)

Is it better to fill missing mac/platform during `override_config_db()` ?

If the original ConfigDB already has mac/platform, we should keep them even we override other fields.

Reply:
Do you mean the generated sysinfo should have less priority than existing one?
Though I think they should be the same.

Reply:
Will submit another PR for sysinfo generation about override.